### ISSUE TITLE
Secrets with a long namespace/name combination are not created

### ISSUE THREAD

**Issue by joggeli34** [external-secrets/external-secrets#2652]
**Describe the bug**
We have secrets in with a long namespace-name together with a long secret name. They are not created after upgrade from 0.9.0 to 0.9.3

**To Reproduce**
Try to create a external secret like the following one with long names:
```yaml
apiVersion: external-secrets.io/v1beta1
kind: ExternalSecret
metadata:
  labels:
    app.kubernetes.io/instance: namespace-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
    app.kubernetes.io/managed-by: Helm
    helm.sh/chart: xxxxx-xxx-2.0.3
  name: external-secret-xxxxxxxx
  namespace: namespace-xxxxxxxxxxxxxxxxxxxxxxxxxx
spec:
  data:
    - remoteRef:
        key: keyname
      secretKey: secretkey
  refreshInterval: 24h0m0s
  secretStoreRef:
    kind: ClusterSecretStore
    name: keystorename
  target:
    name: xxxxx-xxxx-xxxxx-xxxxxxx
```

**Expected behavior**
The secret should be created as it was with version 0.9.0

**Comment by moolen** [comment]
Thanks for reporting! This is a duplicate of #2616. We just released 0.9.4 with a fix for that. Can you upgrade and verify it's working now?

**Comment by joggeli34** [comment]
Sorry I missed the duplicate. I just tried with 0.9.4 and it is working perfectly.
Thanks for the fix
